masq/src/commands/exit_location_command.rs

```diff
@@ -503,11 +502,29 @@ pub mod tests {
         .unwrap_err();
 
         assert!(
-            result.contains(result_expected_one) || result.contains(result_expected_two),
-            "result was {:?}, but expected {:?} or {:?}",
+            result.contains(result_expected),
```

Comment:
Spravne bys zde alespon mel patrat po tom hesle "country-codes", aby bylo jasne, ze ta hlaska z Clap se tyka opravdu toho a ne neceho jineho. 

To se pak dela tak, ze kombinujes dve nebo vice `.contains()`. Jedno obsahuje ten dlouhy text a druha klidne jen ten jeden klicovy termin.

Nezlob se :pray: